#!/usr/bin/python3

def raise_exception_msg(message=&quot;&quot;):
&quot;&quot;&quot;Raise a NameError exception with a message.&quot;&quot;&quot;
raise NameError(message)
